src/github.com/getlantern/gls/gls.go

```diff
@@ -0,0 +1,159 @@
+// Package gls implements goroutine-local storage.
```

Comment:
This is a fork of https://github.com/tylerb/gls.  If we go this route, I'll open a PR upstream.